pkg/controller/servicebinding/servicebinder.go

```diff
@@ -26,13 +26,13 @@ const (
 	// BindingFail binding has failed
 	BindingFail = "BindingFail"
 	//Finalizer annotation used in finalizer steps
-	Finalizer = "finalizer.servicebindingrequest.openshift.io"
+	Finalizer = "finalizer.servicebinding.openshift.io"
```

Comment:
Thoughts on `finalizer.servicebinding.openshift.io` -> `finalizer.servicebinding.operators.coreos.com`?

Reply:
Being fixed in 5f8ae1e , thanks.